hw/syn/data/dc.hjson

```diff
@@ -14,7 +14,7 @@
             {"SV_FLIST"  : "{sv_flist}"}]
 
   // Tool invocation
-  build_cmd:  "dc_shell-xg-t "
+  build_cmd:  "{job_prefix} dc_shell-xg-t "
```

Comment:
sorry i'm a bit out of the loop on the whole flow, but is this job prefix an environment variable we would set..? Or something we pass in as we invoke?

Reply:
This option is needed to run dc in LSF or in docker

Reply:
sorry this is kind of my dumb question..i was asking more if i were to run this, where would i set this option :)

Reply:
you can set that on the command line when running dvsim using the following switch: `--job-prefix` 

we can send you the prefix for our internal infrastructure via PM.